tests/Microsoft.Identity.Web.Test/WebAppExtensionsTests.cs

```diff
@@ -880,7 +880,9 @@ public void PreventChangesInOpenIdConnectOptionsToBeOverlooked()
             // needs to be updated. For this uncomment the 2 lines below, and run the test
             // then diff the files to find what are the new properties
             int numberOfProperties = typeof(OpenIdConnectOptions).GetProperties(System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.Instance).Length;
-#if NET8_0_OR_GREATER
+#if NET9_0_OR_GREATER
+            int expectedNumberOfProperties = 61;
```

Comment:
Do we also need to update the MergedOption method?

Reply:
there is a new member missing in the MergedOptions which needs to be added. could be another PR, please open an issue and add it to our backlog.

Reply:
Created new issue in backlog https://github.com/AzureAD/microsoft-identity-web/issues/2747. Thanks @jennyf19 & @jmprieur